examples/length/length.description

```diff
@@ -0,0 +1 @@
+// Ballerina supports the `length` function that returns the length/size of the given value.
```

Comment:
We need to explain a bit more about what types of values support this function. Otherwise it'll give the wrong impression that this can be called on any type of value.

Reply:
done